sync/sync.go

```diff
@@ -72,7 +72,7 @@ func NewSyncer[H header.Header[H]](
 	var metrics *metrics
 	if params.metrics {
 		var err error
-		metrics, err = newMetrics()
+		metrics, err = newMetrics(params.blockTime)
```

Comment:
We should make it the same as recencyThreshold, which is x2. Ideally, we would make it defined in a single place.